### PR TITLE
Fixes Mobile responsiveness

### PR DESCRIPTION
**Pull request by badrlarhdir**
- [x]  Add a column in admin/DKGTable.tsx and admin/ProxyRow.tsx
- [x]  Fix electionsTable (only showing icons for quickactions in mobile)
- [x]  Election filter in election/Index.tsx reduces z-index
- [x]  Footer UI responsive
- [x]  ElectionID in election/Show.tsx → break-all (for the long ID)

### CODE REVIEW

File: web/backend/config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "SESSION_SECRET": "session secret",
+  "PUBLIC_KEY": "b0e87b48610cd96c3e78ac7b8a217f39a3479941f6728425bad16d62b0c02be5",
+  "PRIVATE_KEY": "983f95ee311e0f545b8bc72f49c0f9709046ff9cdca4cf77141d163b6c43c60f"
+=======
+  "FRONT_END_URL" : "https://localhost:3000",
+  "DELA_NODE_URL" : "localhost:12345",
+  "SESSION_SECRET" : "session secret",
+  "PUBLIC_KEY" : "b0e87b48610cd96c3e78ac7b8a217f39a3479941f6728425bad16d62b0c02be5",
+  "PRIVATE_KEY" : "983f95ee311e0f545b8bc72f49c0f9709046ff9cdca4cf77141d163b6c43c60f"
+>>>>>>> Stashed changes
```
Comment (cmsigrist) [inline web/backend/config.json:14]: Sorry I think I had to stash it to checkout your branch and it didn't merge correctly this file...
@nkcr do we need this file, it has been untracked for some weeks now ?

File: web/frontend/src/mocks/handlers.ts [hunk 2]
```diff
@@ -394,10 +394,9 @@ export const handlers = [
   rest.get(endpoints.getProxiesAddresses, async (req, res, ctx) => {
     await new Promise((r) => setTimeout(r, RESPONSE_TIME));
 
-    const response = [];
-    mockNodeProxyAddresses.forEach((proxy, node) => response.push({ [node]: proxy }));
+    console.log({ Proxies: Object.fromEntries(mockNodeProxyAddresses) });
```
Comment (cmsigrist) [inline web/frontend/src/mocks/handlers.ts:397]: I forgot to remove this log, it's not useful anymore

**Review by cmsigrist — APPROVED** [review]
LGTM ! :D

**Review by nkcr — APPROVED** [review]
looking forward to seeing the result!

**Pull request merged**